code/modules/reagents/reagents/medical.dm

```diff
@@ -668,6 +689,7 @@
 	custom_metabolism = REAGENTS_METABOLISM * 0.25
 
 /datum/reagent/medicine/quickclot/on_mob_life(mob/living/L, metabolism)
+	L.blood_volume = 0.2
```

Comment:
```suggestion
	L.blood_volume += 0.2
```

Reply:
sonovabitch you found it while I found it in testing